charts/opentelemetry-demo/values.yaml

```diff
@@ -500,14 +516,112 @@ opentelemetry-collector:
               allowed_origins:
                 - "http://*"
                 - "https://*"
+      otlp/spanmetrics:
+        protocols:
+          grpc:
+            endpoint: "localhost:65535"
```

Comment:
Shouldn't be needed if `metrics/spanmetrics` pipeline is removed

Reply:
we need the metrics/spanmetrics pipeline